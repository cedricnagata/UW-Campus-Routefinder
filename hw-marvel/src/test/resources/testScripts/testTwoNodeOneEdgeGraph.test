# Test MarvelPaths doesn't allow adding reflexive edge

# Parse through the data and build the graph
LoadGraph reflexiveGraph reflexiveEdge.csv

# List nodes and edges to see if the graph is built correctly
ListNodes reflexiveGraph
ListChildren reflexiveGraph node1
ListChildren reflexiveGraph node2

# Find the shortest path node1 to node2
FindPath reflexiveGraph node1 node2