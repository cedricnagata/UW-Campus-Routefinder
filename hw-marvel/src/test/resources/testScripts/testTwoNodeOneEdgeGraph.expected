# Test MarvelPaths doesn't allow adding reflexive edge

# Parse through the data and build the graph
loaded graph reflexiveGraph

# List nodes and edges to see if the graph is built correctly
reflexiveGraph contains: node1 node2
the children of node1 in reflexiveGraph are: node2(edge1)
the children of node2 in reflexiveGraph are: node1(edge1)

# Find the shortest path node1 to node2
path from node1 to node2:
node1 to node2 via edge1